src/lib.rs

```diff
@@ -223,4 +224,21 @@ impl LepTess {
     ) -> Option<leptonica::Boxa> {
         self.tess_api.get_component_images(level, text_only)
     }
+
+    /// Set the value of an internal Tesseract parameter.
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// let mut lt = leptess::LepTess::new(None, "eng").unwrap();
+    /// lt.set_variable(leptess::Variable::TesseditCharBlacklist, "xyz");
+    /// ```
+    pub fn set_variable(&mut self, name: Variable, value: &str) {
```

Comment:
would be better returning a result from this method to propagate errors instead of hard crash with `unwrap`.

Reply:
Good plan cccbbf67fbdbc6b3f51606a4d95336425288c008